### ISSUE TITLE
The API endpoint is returning Error 500

### ISSUE THREAD

**Issue by krishnapaul242** [krishnapaul242/distronix_front-end-test#1]
The API endpoint is returning Error 500 Internal Server Error upon sending the sample data given in the specification document.

The API endpoint: http://35.197.106.255:3000/api/v1.1/lastMultiple
Request Method: POST
Request Body: 
```                        
                        Type: x-www-form-urlencoded
                        Key: devid
                        Value: d8c68c44e70825b746c80c41add7279a
```
Testing Environment: Postman

**Comment by krishnapaul242** [comment]
Postman is returning an ERROR: "socket hang up"

**Comment by krishnapaul242** [comment]
Update the API endpoint address to the following:
http://nkda.swm.distronix.in:3000/api/v1.1/devstat/lastMultiple

it will fix issue #1.
